src/interface.jl

```diff
@@ -43,6 +43,8 @@ dual_tightening        - whether to use dual tightening techniques (make sure yo
 global_dual_tightening - dual tightening maintained globally valid (when new solutions are found)
 bnb_callback           - an optional callback called at every node of the tree, for example for heuristics
 strong_convexity       - strong convexity of the function, used for tighter dual bound at every node
+sharpness_constant     - the constant M > 0 for (θ, M)-sharpness.
```

Comment:
we should add the expression we use for sharpness (it can differ from one author to the next)

Reply:
I added a short version of the definition that we have in the paper.